var/spack/repos/builtin/packages/py-metpy/package.py

```diff
@@ -22,10 +25,11 @@ class PyMetpy(PythonPackage):
     depends_on('py-importlib-resources@1.3.0:', when='^python@:3.8.999', type=('build', 'run'))
     depends_on('py-matplotlib@2.1.0:', type=('build', 'run'))
     depends_on('py-numpy@1.16.0:', type=('build', 'run'))
-    depends_on('py-pandas@0.22.0:', type=('build', 'run'))
+    depends_on('py-pandas@0.22.0:', when='@1.0', type=('build', 'run'))
+    depends_on('py-pandas@0.24.0:', when='@1.0.1', type=('build', 'run'))
```

Comment:
I just looked at 1.0.1 and the `setup.cfg` still says 0.22.0, is this a mistake?

Reply:
😢 It was a mistake that `setup.cfg` still said that--we had bumped the minimum we were testing to 0.24. I should also note that the bump was motivated by the fact that our minimum supported xarray requires that newer pandas.